src/modules/ttlmonitor/test/TTLMonitorInstrumentTest.cpp

```diff
@@ -156,47 +138,40 @@ TEST_F(TTLMonitorInstrumentTest, Constructor) {
 TEST_F(TTLMonitorInstrumentTest, BeamMonitor) {
   makeHeader(ttlmonitor->ESSReadoutParser.Packet, MonitorReadout);
 
-  ttlmonitor->VMMParser.setMonitor(true);
-  auto Readouts =
-      ttlmonitor->VMMParser.parse(ttlmonitor->ESSReadoutParser.Packet);
-  ASSERT_EQ(Readouts, 9);
+  auto GoodReadouts =
+      ttlmonitor->TTLMonParser.parse(ttlmonitor->ESSReadoutParser.Packet);
+  counters.TTLMonStats = ttlmonitor->TTLMonParser.Stats;
+
+  ASSERT_EQ(GoodReadouts, 5);
+  ASSERT_EQ(counters.TTLMonStats.Readouts, 7);
+  ASSERT_EQ(counters.TTLMonStats.ErrorRing, 1);
+  ASSERT_EQ(counters.TTLMonStats.ErrorFEN, 1);
+  ASSERT_EQ(counters.TTLMonStats.ErrorADC, 1);
+  //ASSERT_EQ(counters.TTLMonStats.ErrorTimeFrac, 1);
```

Comment:
Should this be un-commented or deleted?

Reply:
I'll remove it. Unfortunately I started fixing compile errors and then realised that there were no
ParserTest which should have covered some of these cases.